wm/frame.go

```diff
@@ -563,9 +571,13 @@ func newFrame(c *client) *frame {
 
 	framed.width = w
 	framed.height = h
-	framed.childWidth = attrs.Width
-	framed.childHeight = attrs.Height
-
+	if full || !decorated {
+		framed.childWidth = w
+		framed.childHeight = h
+	} else {
+		framed.childWidth = w - uint16(framed.borderWidth()*2)
```

Comment:
Can you confirm that the previous use of attrs.Width is no longer required here?

Reply:
yes ... attrs.width and attrs.height is not always accurate... i.e. in the case of goland the w/h reported by attrs isn't maximized width and height though it is reporting it wants to be maximized... so basing child width/height off atttrs.width and attrs.height was not accurate.  The more accurate way is to take the width and height we determine for the frame minus the borders/titlebars/decorations.